### PR TITLE
work with serializers, rework models and admin,

### PR DESCRIPTION
**Pull request by airjordan1323**
MUST REWORK IMAGES FOR ADMIN PANEL

### CODE REVIEW

File: basesite/models.py [hunk 1]
```diff
@@ -73,6 +73,11 @@ class Genre(models.Model):
     name = models.CharField("Имя", max_length=100)
     url = models.SlugField(max_length=160, unique=True)
 
+    # def save(self, *args, **kwargs):
+    #     super().save(*args, **kwargs)
+    #     self.url = slugify(self.name, allow_unicode=True) + '-' + str(self.id)
+    #     super().save(update_fields=['url'])
```
Comment (JollyGoal) [inline basesite/models.py]: Why did you commented this out?

File: basesite/models.py [hunk 2]
```diff
@@ -93,6 +98,12 @@ class Meta:
         verbose_name = "Трейлер"
         verbose_name_plural = "Трейлеры"
 
+def __str__(self):
+    return self.number
```
Comment (JollyGoal) [inline basesite/models.py]: What is it for?

File: basesite/models.py (hunk 1 above)
Comment (airjordan1323) [reply basesite/models.py]: because, will been bug with genres, and i off many def's. for understending situation

File: basesite/serializers.py [hunk 3]
```diff
@@ -8,38 +8,41 @@ class Meta:
...
 
+
 class MovieDetailSerializer(serializers.ModelSerializer):
     """ПОЛНЫЙ ФИЛЬМ"""
     category = serializers.SlugRelatedField(slug_field="name", read_only=True)
     directors = serializers.SlugRelatedField(slug_field="name", read_only=True, many=True)
     actors = serializers.SlugRelatedField(slug_field="name", read_only=True, many=True)
-    genres = GenreSerializer()
+    scenario = serializers.SlugRelatedField(slug_field="name", read_only=True, many=True)
+    genres = serializers.SlugRelatedField(slug_field="name", read_only=True, many=True)
```
Comment (JollyGoal) [inline basesite/serializers.py]: URL

**Pull request merged**